tests/test_structure.py

```
@@ -247,6 +250,66 @@ def test_unfolding_prism_folding_edge_cases(self):
         self.assertIsNotNone(up)
 
+    def test_import_fallback_to_lammpsrun_prism(self):
+        structure_path = os.path.abspath(
+            os.path.join(__file__, "..", "..", "src", "lammpsparser", "structure.py")
+        )
+        original_import = __import__
+        import_fail_count = {"lammps": 0}
+        fake_lammpsrun = types.ModuleType("ase.calculators.lammpsrun")
+        fake_lammpsrun.prism = object
+
+        def fake_import(name, globals=None, locals=None, fromlist=(), level=0):
```

Ruff (0.15.14)

[error] Function argument `globals` is shadowing a Python builtin

(A002)